.changeset/friendly-scissors-shop.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/plugin-scaffolder': minor
+---
+
+- **Deprecation** - Deprecated the following exports, please import them directly from `@backstage/plugin-scaffolder-react` instead
+
+  ```
+  createScaffolderLayout
+  ScaffolderLayouts
+  LayoutOptions
+  LayoutTemplate
+  ```
+
+- The following `/alpha` types have removed from this package and moved to the `@backstage/plugin-scaffolder-react/alpha` package
```

Comment:
Is this true?

Reply:
It is.  Why do you ask?

Reply:
Looks like at least `createScaffolderLayout` is `@public` so not available in `/alpha` I think we should just remove the second part of this as they're all available in `@backstage/plugin-scaffolder-react` now right?

Reply:
https://github.com/backstage/backstage/pull/15414/files#diff-30803124f64dd6757ee54c8081a2bb05c00dbbdb1f90fa386203a8c5ddceb9e7R68-R72

Reply:
sorry, my bad.  Should i just remove the second changeset?

Reply:
I didn't mean didn't delete the changeset. 😅  Just the second part 👍

Reply:
> I didn't mean didn't delete the changeset. 😅 Just the second part 👍

lol, reinstated.  I've had too much coffee or not enough